### PR TITLE
BDD: Returning just live clusters

### PR DESCRIPTION
**Pull request by tisnik**
# Description

BDD: Returning just live clusters

## Type of change

- New test scenario added into existing feature file

## Testing steps

N/A

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [ ] new tests can be executed both locally and within docker container

### CODE REVIEW

File: features/DVO_Recommendations/Smart_Proxy_REST_API.feature [hunk 1]
```diff
@@ -26,30 +26,63 @@ Feature: Behaviour specification for new REST API endpoints that will be impleme
...
-                       },
-                       "reports": [
-                           "check": {for example no_anti_affinity"}, // taken from the original full name deploment_validation_operator_no_anti_affinity
-                           "kind": "{kind attribute}",
-                           "description": {description}",
-                       ]
-               ]
-           }
-           """
+     When TEST_USER make HTTP GET request to REST API endpoint /api/v2/namespaces/dvo using his access token
```
Comment (joselsegura) [inline features/DVO_Recommendations/Smart_Proxy_REST_API.feature:36]: Probably endpoint prefix is redundant with the "Gicen" sentence a few lines above

File: features/DVO_Recommendations/Smart_Proxy_REST_API.feature [hunk 2]
```diff
@@ -26,30 +26,63 @@ Feature: Behaviour specification for new REST API endpoints that will be impleme
...
+                      "cluster": {
+                          "uuid": "{cluster UUID}",
+                          "display_name": "{cluster UUID or displayable name}",
+                      },
+                      "namespace": {
+                          "uuid": "{namespace UUID}",
+                          "name": "{namespace real name}", // optional, might be null
+                      },
+                      "reports": [
+                          "check": "{for example no_anti_affinity}", // taken from the original full name deploment_validation_operator_no_anti_affinity
```
Comment (matysek) [inline features/DVO_Recommendations/Smart_Proxy_REST_API.feature:53]: {} is missing here.

Comment (Bee-lee) [reply features/DVO_Recommendations/Smart_Proxy_REST_API.feature:53]: @matysek it's a comment

**Review by Bee-lee — APPROVED** [review]
LGTM overall

File: features/DVO_Recommendations/Smart_Proxy_REST_API.feature (hunk 1 above)
Comment (tisnik) [reply features/DVO_Recommendations/Smart_Proxy_REST_API.feature:36]: aaah, ok :)

Comment (tisnik) [reply features/DVO_Recommendations/Smart_Proxy_REST_API.feature]: fixed

File: features/DVO_Recommendations/Smart_Proxy_REST_API.feature (hunk 2 above)
Comment (tisnik) [reply features/DVO_Recommendations/Smart_Proxy_REST_API.feature]: fixed

**Review by Bee-lee — APPROVED** [review]
still LGTM :)

**Review by matysek — APPROVED** [review]

**Pull request merged**